src/__tests__/parser.test.ts

```diff
@@ -325,6 +325,37 @@ describe('parse', () => {
       ]);
     });
 
+    test('recognizes contentKey comment', () => {
```

Comment:
I added a new `checkDependentValues()` function and updated `parse()` to call it. All it does for now is insure that format `file(json)` files have a `contentKey`, but it could be expanded in the future if we want similar semantics for other values.